tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_data/_templates/layout.html

```diff
@@ -66,10 +66,10 @@
 
     <b>{{project}}</b> {{release}}.<br>
     {% if project.startswith('tk-') %}
-        This documentation is part of the Shotgun Pipeline Toolkit.
+        This documentation is part of the ShotGrid Pipeline Toolkit.
     {% endif %}
     For more information, please visit
-    <a class=custom_post_menu href='https://developer.shotgridsoftware.com'>Shotgun Support</a>.
+    <a class=custom_post_menu href='https://developer.shotgridsoftware.com'>The ShotGrid developer portal.</a>.
```

Comment:
ShotGrid developer documentation?

Reply:
I prefer the term portal as it points to "lots" of documentation.